src/scss/main.scss

```diff
@@ -10,6 +10,8 @@
 // Layout
 
 @import 'layout/header';
+@import 'layout/video';
```

Comment:
the styles for this component should be in the component folder instead of the layout. Please update this import to: `@import 'components/video'`

Reply:
Done.